lib/decidim_staging/budgets_groups_workflow_pam2020.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module DecidimStaging
+  # Specific Workflow for Barcelona's 2020 PAM
+  class BudgetsGroupsWorkflowPam2020 < Decidim::Budgets::Groups::Workflows::Base
+    PAM2020AUTHORIZATIONHANDLER = 'dummy_census_authorization_handler'
+
+    # The budget component in the user's scope is highlighted.
+    def highlighted?(component)
+      component == user_scope_component unless voted?(user_scope_component)
+    end
+
+    # Can vote in the budget component in the user's scope
+    # and in an extra component out of its scope
+    def vote_allowed?(component, consider_progress = true)
+      return true if component == user_scope_component
+
+      extra_components = voted
```

Comment:
the name `extra_components` could be confusing, maybe something like `components_with_order` would be clear?

Reply:
Thanks for the suggestion, I agree.